src/components/notification/notification_event_meta.tsx

```diff
@@ -0,0 +1,144 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { FunctionComponent, useState, ReactNode } from 'react';
+import { EuiIcon, IconType, IconColor } from '../icon';
+import { EuiBadge } from '../badge';
+import { EuiPopover } from '../popover';
+import { EuiButtonIcon } from '../button';
+import { EuiContextMenuPanel, EuiContextMenuPanelProps } from '../context_menu';
+import { EuiI18n } from '../i18n';
+import { EuiNotificationEventReadButton } from './notification_event_read_button';
+import { htmlIdGenerator } from '../../services';
+
+export type EuiNotificationEventMetaProps = {
+  /**
+   * Type of event (e.g. "Alert", "Cloud", etc..). Appears inside a badge.
+   */
+  type: string;
+  /**
+   * Type of severity (e.g. "Critical", "Warning", etc..)
+   */
+  severity?: string;
+  /**
+   * Shows an indicator of the read state of the event. Leave as `undefined` to hide the indicator.
+   */
+  isRead?: boolean | undefined;
+
+  /**
+   * Accepts either our palette colors (primary, secondary ..etc) or a hex value `#FFFFFF`, `#000`.
+   */
+  badgeColor?: IconColor;
```

Comment:
Changed the `healthStatus` to `badgeColor`.

Reply:
`IconColor` might be similar to the badge's color prop, but in order to ensure the correct prop type in case the badge type ever changes would be to use it directly like:
```suggestion
  badgeColor?: EuiBadgeProps['color'];
```